source/administration/changelog.md

```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+ - Added channel restore and channel privacy change endpoints to the local mode using the System bot.
+
+### Bug Fixes
+ - Fixed an issue where users were unable to set a custom status emoji via slash command. Added the logic for detecting unicode emoji and setting it as a custom status emoji via slash commands.
+ - Fixed an issue where messages with fallback text were repeated.
+ - Fixed an issue where a persistent unread badge showed on the **Main Menu** when **Enable Marketplace** or **Enable Plugins** was disabled.
+ - Fixed an issue where sidebar icons were not aligned with the navigator area icons.
+ - Fixed an issue where using CTRL+F in a **Direct Message** channel added the user ID rather than the user's name into the search field.
+ - Fixed an issue where user icons were displayed at full opacity in muted channels.
+ - Fixed an issue where a redundant ``user_update`` websocket event was generated for bot users.
+
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+    - Added ``CollapsedThreads`` to add support for Collapsed Reply Threads (Beta).
+
+#### Database Changes
+ - Removed several redundant Database indexes.
+
+### API Changes
+ - Added a new field, team_id, to the response of ``POST api/v4/groups/{group_id}/channels/{channel_id}/link`` to add a team ID to the response when linking a channel to a group.
+ - Added an optional ``collapsed_threads_supported`` parameter to /channels/members/{userId}/view to indicate that the client supports collapsed threads.
+ - Added an optional ``collapsed_threads_supported`` parameter to /users/{userId}/posts/{postId}/set_unread to indicate that the client supports collapsed threads.
+ - Updated the webapp to pass ``collapsed_threads_supported`` parameters to the server to indicate that the webapp supports collapsed reply threads.
+ - Updated the webapp to correctly mark channels and threads as unread/read when marking root and reply posts as unread/read.
+ - Added a new endpoint ``GET /trial-license/prev`` for fetching last used trial license.
+ - Added two new fields in ``CustomStatus`` struct and modified the APIs to validate and handle them.
+
+### Go Version
+ - v5.37 is built with Go ``1.15.5``.
+
+### Open Source Components
+ - Removed ``reselect`` from https://github.com/mattermost/mattermost-webapp/.
+
+### Known Issues
+ - ``config.json`` can reset when running the command ``systemctl restart mattermost``, and when running any commands that write to the config (e.g. ``config`` or ``plugin``) [MM-33752](https://mattermost.atlassian.net/browse/MM-33752), [MM-32390](https://mattermost.atlassian.net/browse/MM-32390).
```

Comment:
Let's please add a bullet here at the top linking to the CRT doc known issues section